app/controllers/home_controller.rb

```diff
@@ -1,27 +0,0 @@
-class HomeController < ApplicationController
```

Comment:
We shouldn't be removing this behaviour.

Reply:
John was happy with this one. Should be OK.

Reply:
I've asked @nicholsj about this. The dashboard is a quick way for admins to get an overview but will need to be looked at in the future.